package.json

```diff
@@ -17,8 +17,8 @@
   ],
   "scripts": {
     "browser-test": "airtap --local 8080 test/browser*test.js",
-    "test": "standard | snazzy && NODE_OPTIONS='--no-warnings' tap -j 4 --no-cov test/*test.js",
-    "ci": "standard | snazzy && NODE_OPTIONS='--no-warnings' tap -j 4 --cov test/*test.js",
+    "test": "standard | snazzy && NODE_OPTIONS='--no-warnings -r qodaa' tap -j 4 --no-cov test/*test.js",
```

Comment:
I'm not fond of this. My favorite thing about `tap` is that you don't have to use a runner to run the tests; you can simple `node the-test-file.js`. This change breaks that functionality without explaining it to potential new contributors.

I understand this should only be necessary on Node 6. I think we should just drop Node 6 support for pino@5 and prevent this hack.

Reply:
This is a good point, but anyone who is actively developing with Node 6, for whatever reason - has already bought into the fact that newer code may not work or may take extra steps. I think that is going to effect few to no people